### PR TITLE
Allow admin to set test as not counted.

### PR DESCRIPTION
**Pull request by lukas-manduch**
This adds visualization and switch for admin
image
image

### CODE REVIEW

File: cppseminar/testservice/Services/TestRunsService.cs [hunk 1]
```diff
@@ -35,6 +36,19 @@ public async Task<List<TestRun>> GetAsync(string userEmail, string submissionId,
...
+    // Update document. Return null if document doesn't exist else return new document
+    public async Task<TestRun> PatchOneAsync(string userEmail, string testId, BsonDocument update)
+    {
+        var bsonUpdate = new BsonDocument
+        {
+            {"$set" , update}
+        };
+        return await _testRuns.FindOneAndUpdateAsync(
+            x => x.Id == testId && x.CreatedBy == userEmail,
+            bsonUpdate,
```
Comment (koscelansky) [inline cppseminar/testservice/Services/TestRunsService.cs:48]: Why not to inline this?

**Review by koscelansky — COMMENTED** [review]
Just this one nitpick. Otherwise I like it quite a lot. Not firing mongosh every this this will happen is good.

File: cppseminar/testservice/Services/TestRunsService.cs (hunk 1 above)
Comment (lukas-manduch) [reply cppseminar/testservice/Services/TestRunsService.cs]: Fixed

**Comment by koscelansky** [comment]
Just merge it. Looks good.

**Review by koscelansky — APPROVED** [review]

**Pull request merged**